platform/landing-page/pom.xml

```diff
@@ -50,7 +50,7 @@
         <dependency>
             <groupId>org.apache.servicemix.specs</groupId>
             <artifactId>org.apache.servicemix.specs.jsr339-api-2.0</artifactId>
-            <version>2.4.0</version>
+            <version>2.5.0</version>
```

Comment:
We could. I was going back and forth on this because right now those CXF version properties are really only for the features file and we would want to keep them all together. I think we've avoided moving it up to keep people from tightly coupling to the transitive dependencies that CXF is bringing, though it is probably happening anyways. I could really go either way on this if people feel strongly.